### PR TITLE
Add recipe for gretel (an algorithm for the recovery of haplotypes from a metagenome)

### PR DESCRIPTION
**Pull request by johnne**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

gretel relies on hansel (see PR #8651)

### CODE REVIEW

File: recipes/hansel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{% set version = "0.0.81" %}
+
+package:
+  name: hanselx
```
[the remark this answers is not in the document]
Comment (johnne) [reply recipes/hansel/meta.yaml]: I'm using the naming that the creator uses and is available using pip install (see https://github.com/SamStudio8/hansel).

[the remark this answers is not in the document]
Comment (bgruening) [reply recipes/hansel/meta.yaml]: But then the folder name should also be adjusted to be named `hanselx`

[the remark this answers is not in the document]
Comment (johnne) [reply recipes/hansel/meta.yaml]: You're right @bgruening, sorry about that. What would you say is the easiest way for me to change it?

[the remark this answers is not in the document]
Comment (johnne) [reply recipes/hansel/meta.yaml]: Make a new branch 'hanselx' and push the changes to there? Then make a PR for that branch?

[the remark this answers is not in the document]
Comment (bgruening) [reply recipes/hansel/meta.yaml]: You could `git mv hansel hanselx` and you should be fine.

[the remark this answers is not in the document]
Comment (johnne) [reply recipes/hansel/meta.yaml]: @bgruening, merged the upstream master branch so that the directory is now named 'hanselx'

**Pull request merged**